src/Razor/test/testapps/AppWithPackageAndP2PReference/AppWithPackageAndP2PReference.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  
+  <PropertyGroup>
+    <RazorSdkDirectoryRoot>$(RazorSdkArtifactsDirectory)$(Configuration)\sdk-output\</RazorSdkDirectoryRoot>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <RuntimeAdditionalRestoreSources Condition="'$(RuntimeAdditionalRestoreSources)' == ''">$(MSBuildThisFileDirectory)..\TestPackageRestoreSource\</RuntimeAdditionalRestoreSources>
+    <RestoreSources>
+      $(RestoreSources);
+      $(RuntimeAdditionalRestoreSources)
+    </RestoreSources>
+    <ContentVersion>3</ContentVersion>
+
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Avoid referencing the AspNetCore framework since we want the ability to target work-in-progress shims -->
+    <FrameworkReference Remove="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="'$(RunningAsTest)' == ''">
+    <!-- We don't want to run build server when not running as tests. -->
+    <UseRazorBuildServer>false</UseRazorBuildServer>
+  </PropertyGroup>
+
+  <!-- Test Placeholder -->
+
+  <ItemGroup>
+    <PackageReference Include="PackageLibraryDirectDependency" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- During tests the csproj file gets copied to a temporary folder, so we set the reference 
+         relative to $(RuntimeAdditionalRestoreSources) as it simply points to a sibling folder. -->
+    <ProjectReference Include="$(RuntimeAdditionalRestoreSources)\..\ClassLibrary2\ClassLibrary2.csproj" />
```

Comment:
As discussed, this should be a regular project reference. You can use the fixture to do the needful

Reply:
done